src/Text/Parsing/StringParser/String.purs

```diff
@@ -41,6 +44,24 @@ anyDigit = Parser \{ str: str, pos: i } fc sc -> case charAt i str of
   rxDigit :: Rx.Regex
   rxDigit = Rx.regex "^[0-9]" Rx.noFlags
 
+-- | Match single charcter not in the specified array.
+noneOf :: Array Char -> Parser Char
```

Comment:
Also, it would probably be more useful going forward to use `anyChar` and implement a generic `satisfy` function.